core-bundle/tests/DependencyInjection/ConfigurationTest.php

```diff
@@ -180,6 +180,24 @@ public function getReservedImageSizeNames(): \Generator
         yield ['right_bottom'];
     }
 
+    public function testSetsTheConsolePath(): void
```

Comment:
This method can be removed again IMHO. It basically only tests the Symfony config definition processor, which is already tested in their test suite.

Reply:
Yeah I had a typo initially, hence I added a test case :D Sometimes covering the simplest things make sense. But you can also remove it if otherwise you'd merge :)